products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,73 @@
+
+We highly advise you to ask other community members to conduct peer review not only on Pontoon or Locamotion, but on staging. While not all the languages are required for certain projects, each community can opt in the projects at a later time.
+
+## What to Test
+
+### Pre-L10n Test
+* Have your [glossary]((https://transvision.mozfr.org/) available as a reference, select mozilla.org as Repository, your language as your Target Locale
+* Associate the page with a product or site if it is localized. (e.g.: Firefox, Test Pilot)
+* Have the matching US page up as reference, though some strings may not be identical due to A/B testing
+* Have the project you just localized available for editing (Pontoon or Locamotion)
+
+### Linguistic Testing
+* Translation quality in context
+* Grammatically correct in context
+* Check for truncation: button text remains inside the button
+* Header line break wraps at proper place
+* Text not overlapping graphic
+* Terminology is consistent with product, and between web pages
+* Brand names remain in English
+* Product names comply to Mozilla guideline and adhere to what the community has agreed to
+* No corrupted characters
+* Click on the links on the page, which should take you to the pages of the same language if they are localized, or they will be redirected to en-US if the pages are not.  
+* Nav bar terms consistent with the page titles they are linked to (except when Nav bar term is shortened due to space limitation)
+* Footer links don’t overlap with one another
+
+You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.org/projects/mozillaorg/), [Locamotion](https://mozilla.locamotion.org/projects/mozilla_lang/) or [GitHub](https://github.com/mozilla-l10n/www.mozilla.org/).  
+
+### Functionality Testing
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox
+* Font support and readability
+* Footer: verify that the translation of the link is coherent and the link is functional.
+* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/en-US/ to confirm.
+* Deliberately type a wrong broken link, verify whether [404 page](https://www-dev.allizom.org/404/) is in the same language
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team. Indentation and alignment are correct
+
+
+### Testing for Fundraising Campaigns: 
+The annual fundraise campaign is localized in many languages, but not all languages. 
+* Date and time format is country specific
+* Currency availability
+* Postal/zip code in the proper format
+* Mailing address format appropriate for the country
+* Payment system available for the targeted country
+* Country list in your language and order
```

Comment:
The country list is definitely on the form when people fill out personal info. but if this is provided by third party, then we don't have any control. Having said that, the donation process expects donors to understand English ^-^. People will not provide personal info if the process is not in their language, especially with money involved.